config/environment_variables-template.json

```diff
@@ -83,10 +83,10 @@
   "LEGISLATORS_CURRENT_CSV_FILE":   "import_export_theunitedstatesio/import_data/legislators-current.csv",
 
   "_comment":                       "import_export_twitter",
-  "TWITTER_CONSUMER_KEY":           "",
-  "TWITTER_CONSUMER_SECRET":        "",
-  "TWITTER_ACCESS_TOKEN":           "",
-  "TWITTER_ACCESS_TOKEN_SECRET":    "",
+  "TWITTER_CONSUMER_KEY":           "AT1kSVKGXBEkvBZy1CkVqBYfO",
```

Comment:
Hi @abdulrahimq we don't want to add our private keys to our public open source code. :) These variables belong in the "config/environment_variables.json" file which remains private on your machine.